### PR TITLE
Use Terminus to install the plugin

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

**Comment by aaronbauman** [comment]
Thanks for the patch.
This only works for Terminus v3 right?
Do we need to change dependencies in composer.json as well?

**Comment by RobLoach** [comment]
I'm unsure of other versions of Terminus. This is on version 3.0.4. On `self:plugin:list`, it lists...
```
---------------------------- --------------------------------------------------------- ------------------- ---------------- ----------------- 
  Name                         Description                                               Installed Version   Latest Version   Compatible With  
 ---------------------------- --------------------------------------------------------- ------------------- ---------------- ----------------- 
  terminus-config-export       Terminus plugin for syncing Drupal site configuration     1.3                 1.3              ^2.0
```

File: README.md [hunk 1]
```diff
@@ -24,4 +24,6 @@ _NB: be	sure to	use the	`--destination`	flag if	you're not in a	working	drupal e
 
 ## Installation
 
-`composer create-project --no-dev -d ~/.terminus/plugins aaronbauman/terminus-config-export`
```
Comment (aaronbauman) [inline README.md:27]: Maybe just keep both lines for now?
e.g. 

terminus v2: 
`composer create-project`

terminus v3:
`terminus self:plugin:install...`

File: README.md [hunk 2]
```diff
@@ -24,4 +24,6 @@ _NB: be	sure to	use the	`--destination`	flag if	you're not in a	working	drupal e
...
+```
+terminus self:plugin:install aaronbauman/terminus-config-export
```
Comment (RobLoach) [inline README.md:35]:
``` suggestion
# Terminus 2
composer create-project --no-dev -d ~/.terminus/plugins aaronbauman/terminus-config-export

# Terminus 3
terminus self:plugin:install aaronbauman/terminus-config-export
```

File: README.md (hunk 1 above)
Comment (RobLoach) [reply README.md:27]: Great idea! :+1:

File: README.md [hunk 3]
```diff
@@ -24,4 +24,10 @@ _NB: be	sure to	use the	`--destination`	flag if	you're not in a	working	drupal e
 
 ## Installation
 
-`composer create-project --no-dev -d ~/.terminus/plugins aaronbauman/terminus-config-export`
+```
+# Terminus 2
+composer create-project --no-dev -d ~/.terminus/plugins aaronbauman/terminus-config-export
+
+# Terminus 3
+terminus self:plugin:install aaronbauman/terminus-config-export
```
Comment (RobLoach) [inline README.md:32]: Depending on which version of Terminus you have installed, install the plugin by using one of the following commands. You can see which version of Terminus you're using by executing `terminus --version`.

### Terminus 2
```
composer create-project --no-dev -d ~/.terminus/plugins aaronbauman/terminus-config-export
```

### Terminus 3
```
terminus self:plugin:install aaronbauman/terminus-config-export
```

**Pull request merged**